themes/mono/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "mautic/theme-mono",
+  "description": "Mautic Mono Theme",
+  "type": "mautic-theme",
+  "keywords": ["mautic","theme"],
+  "minimum-stability": "dev",
+  "require": {
+    "mautic/core-lib": ">=5.2"
```

```suggestion
    "mautic/core-lib": ">=5.2.2"
```
This theme cannot be used in Mautic 5.2.0 or 5.2.1, as it will result in a 500 error. Since PR #14425 was merged in Mautic 5.2.2, this should be the minimum required version.